environment.yml

```diff
@@ -6,6 +6,6 @@ dependencies:
 - python=3.6.5
 - pip>=18.0
 - conda>=4.5.9
-- patchelf=0.9
+- patchelf=0.9  # comment this out on Mac
```

Comment:
Would it be possible to use the preprocessing selectors here? https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors

Reply:
Reading the link I sent to you more carefully, it makes sense that it doesn't work. Those seem to be metadata only for building packages and not for installing them. I think what you initially had there is good.